config/secrets.yml

```diff
@@ -0,0 +1,22 @@
+# Be sure to restart your server when you modify this file.
+
+# Your secret key is used for verifying the integrity of signed cookies.
+# If you change this key, all old signed cookies will become invalid!
+
+# Make sure the secret is at least 30 characters and all random,
+# no regular words or you'll be exposed to dictionary attacks.
+# You can use `rails secret` to generate a secure secret key.
+
+# Make sure the secrets in this file are kept private
+# if you're sharing your code publicly.
+
+development:
+  secret_key_base: 60daf0d06bb2646d9c60f5036ac363a9a4b48c08f02abe474610e58edfa030f944fa7ab17f7a6b06e26327258ef3727c5f95c9634548f475e1347e5f74d2ffe7
```

Comment:
We're already setting `secret_key_base` in  secret_token.rb (from  #195), and this seems to have a higher priority so I get logged out when I switch between this branch and master. I don't think this would affect production, but could probably just leave this file out since secrets.yml gets deprecated/removed later anyway?